src/openfermion/hamiltonians/_mean_field_dwave_test.py

```diff
@@ -0,0 +1,123 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Tests for mean-field d-wave model module."""
+from __future__ import absolute_import
+
+import unittest
+
+from openfermion.hamiltonians import mean_field_dwave
+
+
+class MeanfieldDwaveTest(unittest.TestCase):
+
+    def setUp(self):
+        self.tunneling = 2.
+        self.sc_gap = 2.
+
+    def test_two_by_two(self):
+
```

Comment:
I know I write this review a lot, but why do we have no project standard on writing one-liners on generically what is being tested in a test.  It's all fine until a test breaks in 6 months and you have to spend all day figuring out what it was supposed to be testing...